pkg/operator/sync.go

```diff
@@ -605,6 +605,15 @@ func (optr *Operator) syncRequiredMachineConfigPools(_ *renderConfig) error {
 				return false, nil
 			}
 
+			updating := isPoolStatusConditionTrue(pool, mcfgv1.MachineConfigPoolUpdating)
+			if updating {
+				glog.Infof("pool %s is still updating, Status: (pool updating: %v total: %d, ready %d, updated: %d, unavailable: %d)", pool.Name, updating, pool.Status.MachineCount, pool.Status.ReadyMachineCount, pool.Status.UpdatedMachineCount, pool.Status.UnavailableMachineCount)
```

Comment:
can we not print status here? This floods machine-config-controller logs whenever a pool is progressing.

Reply:
Agree will remove